docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: ./build
       args:
-        UPSTREAM_VERSION: v23.6.1
+        UPSTREAM_VERSION: v23.11.0
```

Comment:
Please update to [v24.1.1](https://github.com/orgs/gnosischain/packages/container/gnosis-nimbus-eth2/164812888?tag=v24.1.1)